operator/controllers/common/evaluationhandler_test.go

```diff
@@ -229,7 +229,7 @@ func TestEvaluationHandler(t *testing.T) {
 			getSpanCalls:    1,
 			unbindSpanCalls: 1,
 			events: []string{
-				"ReconcileEvaluationSucceeded",
+				"ReconcileEvaluation-Succeeded",
```

Comment:
is this on purpose?

Reply:
The Events were made more readable, but if the team does not agree with this, we can revert it

Reply:
I find this very inconsistent with only subwords split by `-`. I rather have all camelcase or nothing.

Reply:
ok, will revert

Reply:
fixed